examples/tutorials/Differentiation_Infrastructure_in_Deepchem.ipynb

```diff
@@ -0,0 +1,417 @@
+{
```

Comment:
<p>Uncomment Broyden's 2nd method.</p><p>Can you format the roots to render as just 2 numbers? As in [ -0.0459, -0.0663 ]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3912/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>I can do that formatting but i think *grad_fn* is also an important details which could be important to users. So lets keep it.</p>